tests/headers/issue-1554.h

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: --default-enum-style rust_non_exhaustive --raw-line '#![cfg(feature = "nightly")]' --raw-line '#![feature(non_exhaustive)]'
```

Comment:
Now you need to also add `--rust-target nightly` here for CI to pass :)

Reply:
Travis seems to be pleased by https://github.com/rust-lang/rust-bindgen/pull/1575/commits/7ba2cae8c825baf411488ded7d0eb1e6cc17452d